### ISSUE TITLE
Some examples still use python 2

### ISSUE THREAD

**Issue by PTNobel** [cvxpy/cvxpy#2095]
Some of the examples still use python 2; it would be nice to replace them with up to date code!

As an example, https://www.cvxpy.org/examples/applications/nba_ranking.html uses `print` as a statement.

It would be nice to have someone go through examples, test if they run and fix them if they don't. The source code for the examples is mostly in https://github.com/cvxpy/cvxpy/tree/master/doc/source/examples

CC: @phschiele

**Comment by phschiele** [comment]
To make the most out of this effort, we should give some thought to how we want to manage the examples going forward.
At the very least, we should mention something like "last updated using CVXPY 1.3.1".
Ideally, we would further include some automated testing.

**Comment by PTNobel** [comment]
Now that we have backwards compatibility guarantees, we could just make the examples integration tests we have to pass or justify failing before we can make a release.

**Comment by rileyjmurray** [comment]
That sounds reasonable, provided the examples don’t run too long.

**Comment by tschm** [comment]
I would recommend to use only notebooks as examples and integrate them in a jupyterbook. They can then be executed during the compilation of the notebook. Problems can be observed...

**Comment by Transurgeon** [comment]
this issue could be closed I think since there was a PR that addressed it above.
